### ISSUE TITLE
Update conditions to allow remediation request

### ISSUE THREAD

**Issue by Smullz622** [psu-libraries/scholarsphere#1611]
The following conditions need to be added:
- Update `allows_accessibility_remediation_request` so remediation request is allowed if any of the files either failed auto checking or did not go through check due to non-pdf file type. If all files were checked & passed, remediation request should not be allowed. Depends on #1602 
- Instruments should not be allowed to request remediation. On hold until #1595 is merged in

**Comment by Smullz622** [comment]
Closed by #1595
